src/data/support-title.csv

```diff
@@ -65,4 +66,6 @@
 "[花嫁たるもの！！]","[What Makes a Bride!!]"
 "[召しませふぁーすとBite！]","[Try First with Bite!]"
 "[おもい、ねがう]","[A Feeling, a Wish]"
+"[Head-on fight！]","[Head-on fight!]"
 "[ふたつのノーブルライト]","[Two Noble Lights]"
+"[祝福はフーガ]","[Blessing Is a Fugue]"
```

Comment:
@robflop wdyt?

Reply:
I'd add an "a", i.e. "A Blessing Is a Fugue", since otherwise this seems a bit chopped off. 
Alternatively, "Blessings are Fugues", since singular/plural isn't reflected in JP, so it could be either.

2nd Variant with plural sounds better here, i think.